source/production/F0.Minesweeper.Logic.Abstractions/DefaultRandom.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace F0.Minesweeper.Logic.Abstractions
+{
+	public class DefaultRandom : IRandom
```

Comment:
I suggest that we start to declare _public_ types as `sealed` per default, and _unseal_ them when needed.
(Applies to all types in this PR.

Reply:
Done and in mind for the future 👍